### PR TITLE
Created App for NavBar Menu Purposes.  This is for Access OnDemand

### PR DESCRIPTION
**Pull request by gerald-byrket**
Created App for NavBar Menu Purposes.  This is for Access OnDemand

### CODE REVIEW

**Comment by gerald-byrket** [comment]
Please take a close look at the README to ensure I didn't flub anything up.

File: packaging/ondemand-bc_access_menu.spec [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+%setup -q -n %{repo_name}-%{git_tag_minus_v}
+
+
+%build
+
+
+%install
+%__mkdir_p %{buildroot}%{_localstatedir}/www/ood/apps/sys/%{app_name1}
+%__cp -a ./. %{buildroot}%{_localstatedir}/www/ood/apps/sys/%{app_name1}/
+echo v%{version} > %{buildroot}%{_localstatedir}/www/ood/apps/sys/%{app_name1}/VERSION
```
Comment (johrstrom) [inline packaging/ondemand-bc_access_menu.spec:44]: I think a newline in between each app would be helpful to readability. Beyond that, I don't think you want to copy everything (`./.`), you only want to copy the manifest.

```suggestion
%__mkdir_p %{buildroot}%{_localstatedir}/www/ood/apps/sys/%{app_name1}
%__cp -a %{app_name1}/manifest.yml %{buildroot}%{_localstatedir}/www/ood/apps/sys/%{app_name1}/manifest.yml
echo v%{version} > %{buildroot}%{_localstatedir}/www/ood/apps/sys/%{app_name1}/VERSION

```

File: packaging/ondemand-bc_access_menu.spec [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+%global app_name2 bc_access_menu_allocations
+%global app_name3 bc_access_menu_support
+%global app_name4 bc_access_menu_metrics
+
+%{!?package_version: %define package_version %{major}.%{minor}.%{patch}}
+%{!?package_release: %define package_release 1}
+%{!?git_tag: %define git_tag v%{package_version}}
+%define git_tag_minus_v %(echo %{git_tag} | sed -r 's/^v//')
+
+Name:     ondemand-%{app_name1}
```
Comment (treydock) [inline packaging/ondemand-bc_access_menu.spec:14]: It would probably be better to call the RPM `ondemand-bc_access_menu`.

Comment (treydock) [reply packaging/ondemand-bc_access_menu.spec:14]: Ie `ondemand-%{repo_name}`

**Pull request closed without merging**